app/src/main/java/com/eveningoutpost/dexdrip/Models/Libre2Sensor.java

```diff
@@ -0,0 +1,56 @@
+package com.eveningoutpost.dexdrip.Models;
+
+import android.provider.BaseColumns;
+import android.text.format.DateFormat;
+
+import com.activeandroid.annotation.Column;
+import com.activeandroid.annotation.Table;
+import com.activeandroid.query.Select;
+
+import java.sql.SQLException;
+import java.util.List;
+
+@Table(name = "Libre2Sensors", id = BaseColumns._ID)
+public class Libre2Sensor extends PlusModel {
+    static final String TAG = "Libre2Sensor";
+
+    static final String[] schema = {
+            "DROP VIEW IF EXISTS Libre2Sensors;",
```

Comment:
Wont this result in the view being recreated each time the app is reloaded in to memory?

Reply:
Correct. this was an artefact from the startingphase where I designed the View in several ways. Will delete the drop this afternoon.